inspirehep/dojson/common/base.py

```diff
@@ -318,3 +318,27 @@ def deleted_records2marc(self, key, value):
     return {
         'a': inspire_dojson_utils.get_recid_from_ref(value)
     }
+
+
+@hepnames.over('field_categories', '^65017')
+@conferences.over('field_code', '^65017')
+@institutions.over('content_classification', '^65017')
+@jobs.over('research_area', '^65017')
+@experiments.over('field_code', '^65017')
```

Comment:
Yes, I believe the name of the field should be the same in all collections. Maybe `field_category` or `research_area`?